language-support/scala/codegen-testing/src/main/scala/com/digitalasset/ledger/client/binding/encoding/EqualityEncoding.scala

```diff
@@ -103,11 +103,18 @@ object EqualityEncoding extends EqualityEncoding {
       case _ => false
     }
 
-    override implicit def valueMap[A: Fn]: Fn[P.Map[A]] = (a1, a2) => {
+    override implicit def valueMap[A: Fn]: Fn[P.Map[A]] = { (a1, a2) =>
       val ev = implicitly[Fn[A]]
       a1.keys == a2.keys && a1.keys.forall(k => ev(a1(k), a2(k)))
     }
 
+    override implicit def valueGenMap[K: Fn, V: Fn]: Fn[P.GenMap[K, V]] = { (a1, a2) =>
+      // FIXME #2256
```

Comment:
Not sure what I should do here. 
What to do with hashCode ?

Reply:
Probably fine under the constraints I describe at length in previous comment.